.github/workflows/auto-unassign-stale-issues.yml

```diff
@@ -0,0 +1,166 @@
+name: Auto-unassign stale issues
+
+on:
+  schedule:
+    # Run daily at 9 AM UTC
+    - cron: "0 9 * * *"
+  workflow_dispatch:
+  issue_comment: # Allow manual triggers via comments
+    types: [created]
+
+jobs:
+  check-stale-assignments:
+    if: github.repository_owner == 'kubestellar'
+    name: Check and unassign stale issue assignments
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      issues: write
+      pull-requests: read
+    steps:
+      - name: Check out code
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8
+        with:
+          persist-credentials: "false"
+
+      - name: Check stale assignments and send warnings
+        env:
+          GH_TOKEN: ${{ github.token }}
+          REPO: ${{ github.repository }}
+        run: |
+          set -euo pipefail
+
+          # Function to detect active work indicators
+          has_active_work() {
+            local issue_number=$1
+            local assignee=$2
+
+            # Check for recent comments indicating active work (last 7 days)
+            recent_comments=$(gh issue view "$issue_number" --json comments 2>/dev/null | \
+              jq -r '.comments[] | select(.author.login == "'"$assignee"'") | .createdAt' | \
+              tail -5 2>/dev/null || echo "")
+
+            if echo "$recent_comments" | xargs -I {} sh -c '
```

Comment:
If I understand correctly, xargs will succeed if all of the input items cause successful command execution, fail if any one of them fails. I suspect you want the opposite, success if any one of them succeeds.